cdap-watchdog/src/main/java/co/cask/cdap/logging/serialize/LogSerializerUtil.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2014-2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.logging.serialize;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utility functions for serialization.
+ */
+final class LogSerializerUtil {
+
+  private static final String MDC_NULL_KEY = ".null";
+
+  private LogSerializerUtil() {}
```

Comment:
Why?? It is not just for MDC. It just a util for serialization and it would be unwise to be specific, otherwise if there any new util method needed for serialization, yet another class need to be introduced unnecessarily